tests/foreman/cli/test_remoteexecution.py

```diff
@@ -177,8 +200,13 @@ def test_positive_run_default_job_template(self):
 
         @Assert: Verify the job was successfully ran against the host
         """
+        invocation_command = JobInvocation.create({
```

Comment:
Why do we not use factory module for `create()` procedure like we do for all other entities in CLI?